src/main/java/no/fint/consumer/models/dokumentfil/DokumentfilController.java

```diff
@@ -141,7 +143,7 @@ public DokumentfilResources getDokumentfil(
 
 
     @GetMapping("/systemid/{id:.+}")
-    public ResponseEntity getDokumentfilBySystemId(
+    public DokumentfilResource getDokumentfilBySystemId(
```

Comment:
Pay attention to `DokumentfilController` - this one contains manually modified request mappings!

As you might have noticed, there are specs for this that failed during Jenkins build.